File: 백준/Bronze/10886. 0 ＝ not cute ／ 1 ＝ cute/0 ＝ not cute ／ 1 ＝ cute.py
n = int(input())
a = 0
b = 0

for i in range(n):
    j = int(input())
    if j == 1:
        a += 1
    if j == 0:
        b += 1

if a > b:
    print("Junhee is cute!")
else:
    print("Junhee is not cute!")        

File: 백준/Bronze/10886. 0 ＝ not cute ／ 1 ＝ cute/0 ＝ not cute ／ 1 ＝ cute.cc
#include <iostream>
using namespace std;

int main() {
    int n, a, b;
    a = 0;
    b = 0;
    cin >> n;
    int c; 

    for (int i = 0; i < n; i++) {
        cin >> c;
        if (c == 1) {
            a++;
        } else {
            b++;
        }
    }

    if (a > b) {
        cout << "Junhee is cute!";
    } else {
        cout << "Junhee is not cute!";
    }
}